### PR TITLE
Add price oracle projects

### PR DESCRIPTION
**Pull request by rsteimen**
persisting exchange rates on ledger

### CODE REVIEW

File: src/xrpl-cheat-sheet.json [hunk 1]
```diff
@@ -370,6 +370,22 @@
...
         ],
+        "Price Oracles": [
+            {
+                "name": "XRPL Labs Price Aggregator",
+                "link": "https://github.com/XRPL-Labs/XRPL-Persist-Price-Oracle",
+                "description": ""
+            },
+            {
+                "name": "XRP-Oracles",
+                "link": "https://github.com/lathanbritz/XRP-Oracles"
```
Comment (TusharPardhe) [inline src/xrpl-cheat-sheet.json:381]: Add description field here as well

Comment (TusharPardhe) [reply src/xrpl-cheat-sheet.json:381]: Also it'll be great if you add a short description :)

Comment (rsteimen) [reply src/xrpl-cheat-sheet.json:381]: I created "Price Oracles" under "Dev Utilities". Subcategories don't seem to support a description field (no other example).
All three project basically do the same. A description would be identical for all.

Comment (TusharPardhe) [reply src/xrpl-cheat-sheet.json:381]: merging it, I will suggest paraphrasing

**Review by TusharPardhe — APPROVED** [review]

**Pull request merged**